dashboard-widgets.el

```diff
@@ -84,8 +84,12 @@ to the specified width, with aspect ratio preserved."
 (defvar dashboard-banner-logo-title "Welcome to Emacs!"
   "Specify the startup banner.")
 
-(defvar dashboard-init-info (format "%d packages loaded in %s"
-                                    (length package-activated-list) (emacs-init-time))
+(defvar dashboard-init-info
+  ;; Check if package.el was loaded and if package loading was enabled
+  (if (bound-and-true-p package-enable-at-startup)
```

Comment:
This condition is not solid. `package` can be used although `package-enable-at-startup` is nil.